### PR TITLE
Box webhook event payload variants

### PR DESCRIPTION
**Pull request by chinedufn**
Closes https://github.com/XAMPPRocky/octocrab/issues/113

### CODE REVIEW

File: src/models/events.rs [hunk 1]
```diff
@@ -135,35 +135,34 @@ fn deserialize_payload(
...
+            .map(EventPayload::MemberEvent)?,
+        EventType::PullRequestEvent => serde_json::from_value::<Box<PullRequestEventPayload>>(data)
+            .map(|payload| EventPayload::PullRequestEvent(payload))?,
         EventType::PullRequestReviewCommentEvent => {
-            serde_json::from_value::<PullRequestReviewCommentEventPayload>(data)
-                .map(|payload| EventPayload::PullRequestReviewCommentEvent(Box::new(payload)))?
+            serde_json::from_value::<Box<PullRequestReviewCommentEventPayload>>(data)
+                .map(|payload| EventPayload::PullRequestReviewCommentEvent(payload))?
         }
         _ => EventPayload::UnknownEvent(data),
```
Comment (XAMPPRocky) [inline src/models/events.rs:167]:
```suggestion
        _ => EventPayload::UnknownEvent(Box::new(data)),
```

**Review by XAMPPRocky — CHANGES_REQUESTED** [review]
Looks good to me, I have two minor comments that I'd like to address and then we can merge this.

**Comment by chinedufn** [comment]
All feedback addressed and rebased!

**Comment by XAMPPRocky** [comment]
Thank you for your PR, and congrats on your first contribution! 🎉

**Pull request merged**